provider/go.sum

```diff
@@ -1650,6 +1650,7 @@ github.com/spf13/pflag v1.0.5 h1:iy+VFUOCP1a+8yFto/drg2CJ5u0yRoB7fZw3DKv/JXA=
 github.com/spf13/pflag v1.0.5/go.mod h1:McXfInJRrz4CZXVZOBLb0bTZqETkiAhM9Iw0y3An2Bg=
 github.com/spf13/viper v1.4.0/go.mod h1:PTJ7Z/lr49W6bUbkmS1V3by4uWynFiR9p7+dSq/yZzE=
 github.com/spf13/viper v1.7.0/go.mod h1:8WkrPz2fc9jxqZNCJI/76HCieCp4Q8HaLFoCha5qpdg=
+github.com/spotinst/spotinst-sdk-go v1.146.0/go.mod h1:Ku9c4p+kRWnQqmXkzGcTMHLcQKgLHrQZISxeKY7mPqE=
```

Comment:
how is the go.sum changed w/o the go.mod changing?

Reply:
Pretty sure it's because `tfbridge/x` was installed

Reply:
`tfbridge/x` is a package, not a module. It doesn't show up in `go.mod` or `go.sum`. Either way, it has nothing to do with the spotinst go sdk.